File: Buổi 2/BMI.py
height = float(input("Enter your height(cm): "))
while height <= 0:
    height = float(input("Invalid height, please try again: "))
height /= 100
    
mass = float(input("Enter your mass(kg): "))
while mass <= 0:
    mass = float(input("Invalid mass, please try again: "))
    
BMI = float(mass)/(float(height)**2)

print("Your BMI is: ", BMI)

if BMI < 16:
    print("Severely underweight")
elif 16 <= BMI < 18.5:
    print("Underweight")
elif 18.5 <= BMI < 25:
    print("Normal")
elif 25 <= BMI < 30:
    print("Overweight")
elif BMI >= 30:
    print("Obese")

File: Session 6 (C#)/BMI/bmi.cs
using System;

namespace FirstCSProjectọ
{
	class MainClass
	{
		public static void Main(string[] args)
		{
			Console.WriteLine ("Enter your height");
			float height = Convert.ToInt32(Console.ReadLine ());
			height = height / 100;
			while (height <= 0)
			{
				Console.WriteLine ("Invalid height, try again: ");
				height = Convert.ToInt32(Console.ReadLine ());
			}

			Console.WriteLine ("Enter your mass(kg): ");
			float mass = Convert.ToInt32(Console.ReadLine ());
			while (mass <= 0)
			{
				Console.WriteLine("Invalid mass,try again: ");
				mass = Convert.ToInt32(Console.ReadLine ());
			}
			float BMI = mass/(height*height);
			Console.WriteLine("Your BMI is: " + BMI);

			if (BMI < 16) {
				Console.WriteLine ("Severely underweight");
			}
					else if (BMI < 18.5)
					{
						Console.WriteLine("Underweight");
					}
					else if (BMI < 25)
					{
						Console.WriteLine("Normal");
					}
					else if (BMI < 30)
					{
						Console.WriteLine("Overweight");
					}
					else if (BMI >= 30)
					{
						Console.WriteLine("Obese");
					}
		
		}
	}
}

